3rdparty/python/README.md

```diff
@@ -0,0 +1,9 @@
+# requirements.txt
+Declare all your dependencies.
+Ideally, we'd pin everything in here, but it's not the end of the universe if a few are unpinned.
```

Comment:
Things should probably only be pinned in here if they need to be for some reason (and that reason should be documented in a comment).

Reply:
updating readme